packages/components/tests/dummy/app/templates/components/side-nav.hbs

```diff
@@ -0,0 +1,144 @@
+
+  {{! ADD YOUR CONTENT HERE }}
+
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-side-nav">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-side-nav">
+
+    {{! ADD YOUR CONTENT HERE }}
+
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  {{! ADD YOUR CONTENT HERE }}
+
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="design-guidelines"><a href="#design-guidelines" class="dummy-link-section">§</a>
+    Design guidelines</h3>
+  {{! UNCOMMENT THIS BLOCK (once the link and/or the image are available) }}
+  {{!
+  <div class="dummy-design-guidelines">
+    <p class="dummy-paragraph">
+      <a href="[ADD THE LINK TO THE FIGMA FILE/PAGE HERE!]" target="_blank" rel="noopener noreferrer">Figma UI Kit</a>
+    </p>
+    <br />
+    <img class="dummy-figma-docs" src="/assets/images/side-nav-design-usage.png" alt="" role="none" />
+  </div>
+  }}
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
+
+  {{! ADD YOUR CONTENT HERE }}
+
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
+
+  {{! WIP: New SideNav component }}
+  <Hds::SideNav>
+    <:header>
+      <Hds::SideNav::HomeLink @icon="hashicorp" @text="HashiCorp" @href="#" />
+
+      <Hds::Dropdown as |dd|>
+        <dd.ToggleIcon @icon="help" @text="settings menu" />
+        <dd.Title @text="Signed In" />
+        <dd.Description @text="email@domain.com" />
+        <dd.Separator />
+        <dd.Interactive @route="components" @text="Settings and Preferences" />
+      </Hds::Dropdown>
+
+      <Hds::Dropdown as |dd|>
+        <dd.ToggleIcon @icon="user" @text="user menu" />
+        <dd.Title @text="Signed In" />
+        <dd.Description @text="email@domain.com" />
+        <dd.Interactive @route="components" @text="Account Settings" />
+      </Hds::Dropdown>
+    </:header>
+
+    <:body>
+      <Hds::SideNav::Section as |S|>
+        <S.Item @icon="dashboard">Dashboard</S.Item>
+      </Hds::SideNav::Section>
+
+      <Hds::SideNav::Section @header="Services" as |S|>
+        <S.Item @label="Boundary" @icon="boundary" @href="#" />
+        <S.Item @label="Consul" @icon="consul" @href="#" />
+        <S.Item @label="Packer" @icon="packer" @href="#" />
+        <S.Item @label="Vault" @icon="vault" @href="#" />
+        <S.Item @label="Vault Secrets" @icon="lock" @href="#" />
+        <S.Item @label="Terraform" @icon="terraform" @href="#" />
+        <S.Item @label="Vagrant" @icon="vagrant" @badge="Alpha" @href="#" />
+
+        {{! TODO: Waypoint should have nested links: }}
```

Comment:
Below are different ideas of what I was considering. I think it probably makes snese to have 2 different types of Items/Link components. One component for a simple link and a different one for a nested link.

I'm not sure yet how the nested link component should work exactly though.

Reply:
I am not sure where this is happening in Cloud UI, can you point to a specific page (or screenshot) or code in their codebase where this is a real case? seems so weird to me that they implemented it this way (with nested links)

Reply:
Cloud ui code isn't set up as a reusable component. I'll look at their code more closely again.

Reply:
I'll just start over on this part. I was trying to come up with an API that would be more easy to use and reusable.